giraffe/src/utils/fromFlux.ts

```diff
@@ -189,6 +189,151 @@ export const fromFlux = (fluxCSV: string): FromFluxResult => {
   }
 }
 
+export const fastFromFlux = (fluxCSV: string): FromFluxResult => {
+  const columns: Columns = {}
+  const fluxGroupKeyUnion = new Set<string>()
+  const resultColumnNames = new Set<string>()
+  let tableLength = 0
+
+  try {
+    const chunks = splitChunksFast(fluxCSV)
+
+    // declaring all nested variables here to reduce memory drain
+    let tableText = ''
+    let tableData: any = []
+    let annotationText = ''
+    let columnType: any = ''
+    let columnKey = ''
+    let columnDefault: any = ''
+
+    for (const chunk of chunks) {
+      const splittedChunk = chunk.split('\n')
+
+      const tableTexts = []
+      const annotationTexts = []
+
+      splittedChunk.forEach(line => {
+        if (line.startsWith('#')) {
+          annotationTexts.push(line)
+        } else {
+          tableTexts.push(line)
+        }
+      })
+
+      tableText = tableTexts.join('\n').trim()
```

Comment:
duplicated data by splitting 1000 lines as a string into 1000 objects containing a string, just to join the 1000 lines into a string again